data/newMigrations/017-races.js

```diff
@@ -0,0 +1,10 @@
+exports.up = (knex) => {
+    return knex.schema.createTable('races', function (tbl) {
+        tbl.increments('race_id').primary();
```

Comment:
`.primary()` is not needed, when using `.increments()` knex sets it as primary behind the scenes